recipes/gnuplot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set pkgname = "gnuplot" %}
+{% set version = "5.0.4" %}
+{% set sha256 = "151cb845728bde75eb9d1561b35140114a05a7c52a52bd35b4b2b3d944e0c31e" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/{{ pkgname }}/files/{{ pkgname }}/{{ version }}/{{ pkgname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - ncurses ==5.9
+    - readline ==6.2
+    - libgd
+    - cairo
+    - qt
+  run:
+    - ncurses ==5.9
+    - readline ==6.2
+    - libgd
+    - cairo
+    - qt
+
+test:
+  files:
+    - test-data.txt
+  commands:
+    - gnuplot -e "set terminal dumb; set style histogram; p 'test-data.txt'"
+    - gnuplot -e "set terminal png"
+
+about:
+  home: https://github.com/gnuplot/gnuplot
+  license: Gnuplot License
```

Comment:
Apparently some files are dual-licensed. Would be great to figure out what those other licenses are and add them here. Often Linux distros do a pretty good job of this stuff. So checking a few Linux distros may give us a good idea of what is going on.